### PR TITLE
Adds CogStation for testing.

### PR DESCRIPTION
**Pull request by EmeraldSundisk**
## About The Pull Request

**This pull request is effectively the pre-release of CogStation**, a remake of Goonstation's Cogmap1 for TGcode servers. Originally started by someone who wants to remain anonymous for all I know, I've gone ahead and seen to it their work was finished.

freesample

It isn't a perfect 1-to-1 remake, partially due to differences in server code in addition to feedback from the Citadel community. However, CogStation seeks to capture the spirit of the original while featuring its own unique charm. **Most of its defining features have been preserved, including:**

- **Integrated Artificial Intelligence:** Unlike most maps that relegate it off to a satellite, CogStation's AI is more or less the heart of the station.
- **Thermo-Electric Generator:** For many engineers, the TEG is usually a side project that comes after the supermatter engine is fired up, but on CogStation it's your primary power source.
- **Advanced Disposal Systems:** CogStation features independent disposal pipe systems for trash, mail, even corpses!
- **Routing System:** A unique method of cargo delivery that lets you launch goods through space. (It isn't fully functional yet, unfortunately.)

**It's also undergone some modifications in order to better suit TGcode, such as:**

- Pod bays have been replaced with conventional shuttle docks, making the map Lavaland-ready.
- The Research Sector features a newly-added Xenobiology Lab and Toxins test range.
- A "Discount Transfer Centre" has been set up next to Security, if perma isn't enough.
- You'll find a complete telecomms satellite outside the bridge, as opposed to merely a receiver.

**To top it all off, CogStation comes with new arrival and escape shuttles.** There's even a new "white ship" for those willing to go look for it!

## Why It's Good For The Game

In addition to being a new choice in map, CogStation seeks to provide a unique experience. Much of modern Space Station 13 can trace itself back to Goon, and this remake is intended as a homage.

## Changelog
:cl:
add: Adds CogStation to the map rotation.
add: Introduces the CogStation arrivals shuttle, "NES Classic" escape shuttle, and the CogStation "White Ship".
tweak: Adds several new area designations to accommodate CogStation.
/:cl:

### CODE REVIEW

**Comment by Detective-Google** [comment]
coggers

**Comment by EmeraldSundisk** [comment]
Cogchamp.

**Comment by Xantholne** [comment]
> coggers

how dare you steal my line.

Also can we get a map image if possible?

**Comment by EmeraldSundisk** [comment]
I'm not sure about the whole thing at once, unfortunately. But lemme see.

**Comment by Xantholne** [comment]
I think the only easiest way is from dreammaker, not 100% certain

**Comment by EmeraldSundisk** [comment]
The issue is I'm not sure if I can zoom out enough to do it justice.

**Comment by EmeraldSundisk** [comment]
Ah, might need to run it through the mapmerge tool.

**Comment by EmeraldSundisk** [comment]
Or I would, if it worked on this computer. Gimme a bit.

**Comment by EmeraldSundisk** [comment]
Hmmm...going to take another stab at it.

File: tgstation.dme [hunk 1]
```diff
@@ -15,6 +15,10 @@
 
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
+#include "_maps\cogstation.dm"
+#include "_maps\shuttles\arrival_cog.dmm"
+#include "_maps\shuttles\arrival_cogstation.dmm"
+#include "_maps\shuttles\whiteship_cog.dmm"
```
Comment (Ghommie) [inline tgstation.dme]: No, don't do this.

File: code/modules/paperwork/paper_premade.dm [hunk 2]
```diff
@@ -1,76 +1,178 @@
...
-	info = "<BR>CentCom Security<BR>Port Division<BR>Official Bulletin<BR><BR>Inspector,<BR>There is an emergency shuttle arriving today.<BR><BR>Approval is restricted to Nanotrasen employees only. Deny all other entrants.<BR><BR>CentCom Port Commissioner"
-
-
-///////////	Lavaland
-
-/obj/item/paper/fluff/stations/lavaland/orm_notice
-	name = "URGENT!"
-	info = "A hastily written note has been scribbled here... <br><br> Please use the ore redemption machine in the cargo office for smelting. PLEASE! <br><br>--The Research Staff"
-
+/*
```
Comment (Ghommie) [inline code/modules/paperwork/paper_premade.dm]: Damn line endings, this is why I tell you to use git instead of uploading your own files.

**Review by Ghommie — CHANGES_REQUESTED** [review]
And convert the maps to the tgm format.

File: tgstation.dme (hunk 1 above)
Comment (EmeraldSundisk) [reply tgstation.dme]: Should be converted.

File: code/modules/paperwork/paper_premade.dm (hunk 2 above)
Comment (EmeraldSundisk) [reply code/modules/paperwork/paper_premade.dm]: I'm sorry about being captain shitcode but it was all I was able to do quickly. If this doesn't work, I'll have to revisit it later today.

**Comment by Detective-Google** [comment]
@EmeraldSundisk affectedarc(think tha'ts the name don't remember) hosts a map render hosting thing, might look into that.

File: tgstation.dme (hunk 1 above)
Comment (Detective-Google) [reply tgstation.dme]: map files aren't supposed to be checked or it force loads them, they get loaded afterwards is what he's saying

**Comment by Xantholne** [comment]
@AffectedArc07 is the one

**Comment by Iamgoofball** [comment]
YOOOOOOOOOOOO IM STEALING THIS FOR TG

**Comment by Ghommie** [comment]
> 
> 
> YOOOOOOOOOOOO IM STEALING THIS FOR TG

It's free AGPL real estate.

File: tgstation.dme (hunk 1 above)
Comment (EmeraldSundisk) [reply tgstation.dme]: This should fix that!

Comment (EmeraldSundisk) [reply tgstation.dme]: Um, hang on, actually...

Comment (EmeraldSundisk) [reply tgstation.dme]: Okay, nevermind. How?

Comment (EmeraldSundisk) [reply tgstation.dme]: NOW it should be fixed.

File: code/modules/paperwork/paper_premade.dm (hunk 2 above)
Comment (EmeraldSundisk) [reply code/modules/paperwork/paper_premade.dm]: Should be sorted now.

**Comment by EmeraldSundisk** [comment]
I don't have an answer to what's going on with all of this other stuff.

**Comment by Iamgoofball** [comment]
Pull out your map, make a new PR on a fresh branch rebased off of master.

**Comment by EmeraldSundisk** [comment]
Round...two?

**Comment by EmeraldSundisk** [comment]
Working on getting the prison shuttle working.

**Pull request closed without merging**